span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/SpanNormalizer.java

```diff
@@ -77,7 +80,12 @@ public StreamsBuilder buildTopology(
                 jobConfig,
                 (traceid, span) -> traceid.getTenantId(),
                 new KeyHashPartitioner<>(),
-                new GrpcChannelRegistry());
+                new GrpcChannelRegistry(
```

Comment:
I guess we have the same issue in all of these kafka stream apps - not only is the channel registry not provided the framework (as we discussed offline), but worse, each of these apps is not actually managing the channel registry, so when the app is shutdown the channel registry is not (which prevents the channels from being closed and releasing the http connection).

Reply:
fixed - used channel-registry from kafka-streams-framework